### ISSUE TITLE
Feature request: Add flag to show stock number

### ISSUE THREAD

**Issue by vic18t** [ianmarmour/nvidia-clerk#61]
I recall in the first version that the stock number was displayed. This would cause errors and delays, but I really think it was a cool feature. 

Can this be added back in as a command line option (does not loop trying to order item, just checks stock number once)?

**Comment by javitab** [comment]
I'll second that idea. I was literally just thinking about that, was potentially going to run the old version adhoc just to get a sense of what it's still looking like.

**Comment by vic18t** [comment]
Yeah, just run v.06

Apparently nVidia has not added any stock to 3080 since after launch.

**Comment by vic18t** [comment]
@ianmarmour  This isn’t possible anymore through the API. However, someone posted on reddit a URI that shows a server response with SKU’s meta flags - including an “in-stock” true/false flag. Maybe something to look into?

**Comment by javitab** [comment]
> @ianmarmour This isn’t possible anymore through the API. However, someone posted on reddit a URI that shows a server response with SKU’s meta flags - including an “in-stock” true/false flag. Maybe something to look into?

Can you link to that?

**Comment by vic18t** [comment]
"inventoryStatus": { "uri": "https://dispatch-nvidia.digitalriver.com/v1/shoppers/me/products/5438481700/inventory-status", "availableQuantityIsEstimated": "false", "productIsInStock": "true", "productIsAllowsBackorders": "false", "productIsTracked": "true", "requestedQuantityAvailable": "true", "status": "PRODUCT_INVENTORY_OUT_OF_STOCK", "statusIsEstimated": "false" }

Is productIsInStock flag available to see still?

**Comment by ianmarmour** [comment]
Yes but product_inventory_out_of_stock is what we use because it's what nvidia uses to turn on and off sales.

**Comment by vic18t** [comment]
> Yes but product_inventory_out_of_stock is what we use because it's what nvidia uses to turn on and off sales.

Yes absolutely, use product_inventory_out_of_stock to see if ATC is on or off. But also displaying productIsInStock is very useful as they might turn on product_inventory_out_of_stock soon.

**Comment by alopex92** [comment]
seems there is a security rule blocking me to test that uri ( https://dispatch-nvidia.digitalriver.com/v1/shoppers/me/products/5438481700/inventory-status )... is it just me or a global rule?

**Comment by redstone2019** [comment]
global
